packages/snaps-controllers/coverage.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Did you confirm that `jest-it-up` still doesn't allow decreasing coverage with those?

Reply:
We're not using `jest-it-up` anymore, as this would update the coverage numbers for the Jest config only, and cause Jest to fail if the coverage collected by Jest only is not high enough.

The new script checks if coverage decreases and fails with exit code 1 if so.